gflownet/policy/cnn.py

```diff
@@ -0,0 +1,91 @@
+import torch
+from omegaconf import OmegaConf
+from torch import nn
+
+from gflownet.policy.base import Policy
+
+
+class CNNPolicy(Policy):
+    def __init__(self, config, env, device, float_precision, base=None):
+        self.env = env
+        super().__init__(
+            config=config,
+            env=env,
+            device=device,
+            float_precision=float_precision,
+            base=base,
+        )
+        self.is_model = True
```

Comment:
If I understand correctly,`self.is_model` should be True only if the policy type is neither fixed or uniform. So, maybe it's better to move `self.is_model = True` to the instantiate method after the cnn model is created

Reply:
will be resolved in next commit, thanks